CHANGELOG.md

```diff
@@ -9,16 +10,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Initial open-source implementation ([#1])
+- Annotate all alerts with the Syn component name
+- Allow configuring custom annotations via inventory
 
 ### Changed
+
 - Upgrade kube-prometheus ([#6])
 
 ### Fixed
 
 - Update kube-prometheus version locks to include recent dependency fixes ([#4]).
 
-[Unreleased]: https://github.com/projectsyn/component-rancher-monitoring/compare/084a263baf909b627d2861790806ac8f7de3f580...HEAD
-
+[unreleased]: https://github.com/projectsyn/component-rancher-monitoring/compare/084a263baf909b627d2861790806ac8f7de3f580...HEAD
```

Comment:
Not sure if this is case sensitive or not...

Reply:
Unrelated change by my autoformatter 🙈  will revert